### PR TITLE
Fix 560

### PR DESCRIPTION
**Pull request by emmetthitz**
Fixes for #560 without build changes, as per @dkoshkin's request.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,4 @@ integration/vendor
 vendor-provision/*
 vendor-kuberang/*
 vendor-helm
+.glide
```
[the remark this answers is not in the document]
Comment (emmetthitz) [reply .gitignore]: Sure thing.

File: Makefile [hunk 2]
```diff
@@ -15,8 +15,8 @@ HOST_GOARCH = $(shell go env GOARCH)
 # Versions of external dependencies
 GLIDE_VERSION = v0.11.1
 ANSIBLE_VERSION = 2.3.0.0
-PROVISIONER_VERSION = v1.6.0
-KUBERANG_VERSION = v1.2.1
+PROVISIONER_VERSION = v1.6.1
+KUBERANG_VERSION = v1.2.2
```
Comment (dkoshkin) [inline Makefile]: Something weird is happening here, this is already at `v1.6.1` and `v1.2.2`

File: ansible/roles/kube-dns/tasks/main.yaml [hunk 3]
```diff
@@ -23,9 +23,15 @@
     - name: fail if DNS pod validation command returned an error
       fail:
         msg: |
-          Attempted to validate the DNS pods, but got an error from kubectl: {{ readyReplicas.stderr }}
+          Attempted to validate the DNS pods, but got an error: {{ readyReplicas.stderr }}
```
Comment (dkoshkin) [inline ansible/roles/kube-dns/tasks/main.yaml]: Merge error?

File: ansible/roles/kube-dns/tasks/main.yaml [hunk 4]
```diff
@@ -23,9 +23,15 @@
...
       when: readyReplicas.stderr != ""
 
+    - name: fail if DNS pod validation command could not determine if the DNS pods are ready
```
Comment (dkoshkin) [inline ansible/roles/kube-dns/tasks/main.yaml]: Merge error?

File: ansible/roles/kube-dns/tasks/main.yaml [hunk 5]
```diff
@@ -35,6 +41,12 @@
       register: failedDNSPodNames
       when: readyReplicas.stdout|int != kube_dns_replicas|int
 
+    - name: fail if DNS pod validation command could not determine the broken pod
```
Comment (dkoshkin) [inline ansible/roles/kube-dns/tasks/main.yaml]: Merge error?

File: ansible/roles/kube-dns/tasks/main.yaml (hunk 3 above)
Comment (emmetthitz) [reply ansible/roles/kube-dns/tasks/main.yaml]: Hair pulling

**Pull request closed without merging**